index.html

```diff
@@ -16,11 +16,44 @@
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
-  </header>
   <main>
+    <div class="heading">
+      <header class="container">
+        Code Journal
+      </header>
+    </div>
```

Comment:
You should apply the background color to `header`, and give that element a child with the `container` class. Within that container, make a `row` and give that `row` a child with a `column-` class, then us an h1-h6 element for this text. That should vastly reduce the amount of special styling you need to apply to the `heading` class here.